src/main.rs

```diff
@@ -32,10 +32,12 @@ const VERSION: &str = env!("CARGO_PKG_VERSION");
 // Defaults
 const DEFAULT_LOG_LEVEL: LevelFilter = LevelFilter::Debug;
 const DEFAULT_THRESHOLD: &str = "10 GB";
+const DEFAULT_DELETION_CHUNK_SIZE: usize = 1;
 
 // Command-line argument and option names
 const THRESHOLD_OPTION: &str = "threshold";
 const KEEP_OPTION: &str = "keep";
+const DELETION_CHUNK_SIZE_OPTION: &str = "deletion-chunk-size";
```

Comment:
I know these 3 constants weren't sorted before, but would you mind sorting them while you're here? So it would be `DELETION_CHUNK_SIZE_OPTION`, then `KEEP_OPTION`, then `THRESHOLD_OPTION`.